pkg/genericcontrolplane/server.go

```diff
@@ -238,6 +238,9 @@ func BuildGenericConfig(
 	// on a fast local network
 	genericConfig.LoopbackClientConfig.DisableCompression = true
 
+	// Accept both protobuf and application/json for loopback client.
+	genericConfig.LoopbackClientConfig.AcceptContentTypes = "application/vnd.kubernetes.protobuf,application/json"
```

Comment:
does CRDs support protobuf ? I think that the loopback client mostly deals with CRDs resources.

Reply:
This should allow loopback to use proto for built-in types, and fall back to json for CRDs